.github/workflows/syft-version_tests.yml

```diff
@@ -12,6 +12,8 @@ on:
   pull_request:
     branches:
       - dev
+    paths:
+      - "packages/syft/tests/trigger/version_tests"
```

Comment:
This is to run the tests for each python version?

Reply:
This test run all the versions of torch, python in a combinatorial factor,
We have run it in a nightly test, If you would like to run it in a PR, you have to modify that particular or you could run it manually.
I modified it like  this way  before,Due to lot of PR yesterday, this got deleted by mistake.